### DISCUSSION TITLE
隐藏记笔记 功能优化建议

### DISCUSSION THREAD

**Discussion by pinaZC** [the1812/Bilibili-Evolved#4285] [功能建议 / Ideas]
看了下原建议是因为美观和常用性问题，所以建议隐藏按钮https://github.com/the1812/Bilibili-Evolved/discussions/4124#discussion-5038093
但是目前达成的效果是，评论区的所有笔记也无法查看（点击展开没有弹窗，没有任何反应）
![image](https://github.com/the1812/Bilibili-Evolved/assets/72682556/bc787f45-6024-4bb6-8693-6aa61c6e1f73)
目前评论区有越来越多发笔记的用户，一刀切体验太差了
建议功能改为只隐藏播放器右下的记笔记按钮
最好的效果还是这两个按钮都移到旁边的···里（目前里面只有一个官方功能）
![image](https://github.com/the1812/Bilibili-Evolved/assets/72682556/18893fd6-cd4a-4f62-a3ed-725bdfee1286)
望采纳，谢谢大佬

**Comment by the1812** [comment]
有视频链接吗, 可展开的笔记不知道该去哪找了

**Reply by Geeyun-JY3** [reply]
[下播型泛式录播组](https://space.bilibili.com/281341996)的视频就不少用笔记功能的打轴的轴 man，都是可展开的笔记，如：https://www.bilibili.com/video/BV1Tr4y1Z76w/

**Comment by Geeyun-JY3** [comment]
b 站是最近抽风了还是故意的，评论区里的长笔记的`展开`按钮直接没了：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/cc7a5bfc-52bb-4ab2-909b-442a336d9375)
